gbus/worker.go

```diff
@@ -473,6 +485,58 @@ func (worker *worker) invokeHandlers(sctx context.Context, handlers []MessageHan
 		))
 }
 
+func (worker *worker) handleDuplicates(message *BusMessage, delivery amqp.Delivery, msgSpecificLogEntry *logrus.Entry) (bool, error) {
+	if worker.delicatePolicy == DeduplicationPolicyNone {
+		return false, nil
+	}
+	duplicate, err := worker.duplicateStore.MessageExists(message.IdempotencyKey)
```

Comment:
shouldn't we be passing in the active transaction ?

Reply:
For message exists there is no real need, we can and we pass in the `StoreMessage` but here it brings no to little value